data/SiteConfig.js

```diff
@@ -1,65 +1,71 @@
 const config = {
-  siteTitle: "The JAC", // Site title.
-  siteTitleShort: "The JAC", // Short site title for homescreen (PWA). Preferably should be under 12 characters to prevent truncation.
+  siteTitle: 'The JAC', // Site title.
+  siteTitleShort: 'The JAC', // Short site title for homescreen (PWA). Preferably should be under 12 characters to prevent truncation.
   siteTitleAlt: "The JAC's new CD: A Gathering", // Alternative site title for SEO.
-  siteLogo: "/logos/logo-1024.png", // Logo used for SEO and manifest.
-  siteUrl: "https://the-jac.com", // Domain of your website without pathPrefix.
-  pathPrefix: "/", // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
+  siteLogo: '/logos/logo-1024.png', // Logo used for SEO and manifest.
+  siteUrl: 'https://the-jac.com', // Domain of your website without pathPrefix.
+  pathPrefix: '/', // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
   siteDescription: "A site for The JAC's new CD: A Gathering. Now available.", // Website description used for RSS feeds/meta description tag.
-  siteRss: "/rss.xml", // Path to the RSS file.
-  siteRssTitle: "The JAC RSS feed", // Title of the RSS feed
-  siteFBAppID: "1825356251115265", // FB Application ID for using app insights
-  googleAnalyticsID: "UA-47311644-5", // GA tracking ID.
-  disqusShortname: "https-the-jac-com", // Disqus shortname.
-  dateFromFormat: "YYYY-MM-DD", // Date format used in the frontmatter.
-  dateFormat: "DD/MM/YYYY", // Date format for display.
-  postsPerPage: 4, // Amount of posts displayed per listing page.
-  userName: "Advanced User", // Username to display in the author segment.
-  userEmail: "AdvancedUser@example.com", // Email used for RSS feed's author segment
-  userTwitter: "", // Optionally renders "Follow Me" in the UserInfo segment.
-  userLocation: "North Pole, Earth", // User location to display in the author segment.
-  userAvatar: "https://api.adorable.io/avatars/150/test.png", // User avatar to display in the author segment.
+  siteRss: '/rss.xml', // Path to the RSS file.
+  siteRssTitle: 'The JAC RSS feed', // Title of the RSS feed
+  siteFBAppID: '1825356251115265', // FB Application ID for using app insights
+  googleAnalyticsID: 'UA-47311644-5', // GA tracking ID.
+  disqusShortname: 'https-the-jac-com', // Disqus shortname.
+  dateFromFormat: 'YYYY-MM-DD', // Date format used in the frontmatter.
+  dateFormat: 'DD/MM/YYYY', // Date format for display.
+  // postsPerPage: 4, // Amount of posts displayed per listing page.
```

Comment:
Disabled this so that `landing` template rendered be default (where I put the `EmbeddedContent`) as I didn't want to get into the `listing` one. Can change back before merge.